conf/messages.en

```diff
@@ -41,19 +41,20 @@ footer.links.terms_and_conditions.text=Terms and conditions
 
 # Compliance statement
 # ----------------------------------------------------------
-accessibility.title=Accessibility statement for External Guidance - to Be Defined
+accessibility.title=Accessibility statement for HMRC Interactive Guidance
 
 ## section one
-compliant.one.heading=Accessibility statement for External Guidance - to Be Defined
+compliant.one.heading=Accessibility statement for HMRC Interactive Guidance
 compliant.one.lede=This accessibility statement explains how accessible this service is, what to do if you have difficulty using it, and how to report accessibility problems with the service.
 compliant.one.p1.1=This service is part of the wider GOV.UK website. There is a separate
 compliant.one.p1.2=accessibility statement
 compliant.one.p1.3=for the main GOV.UK website.
-compliant.one.p2=This page only contains information about the External Guidance service, available at www<to be defined>/guidance
+compliant.one.p2.1=This page only contains information about the External Guidance service, available at
```

Comment:
This should be "This page only contains information about HMRC interactive guidance available at"